### PR TITLE
fix carbon

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/nestjs/docs.nestjs.com/issues/1790

### CODE REVIEW

**Comment by Arhell** [comment]
cc @kamilmysliwiec

File: src/app/homepage/homepage.component.ts [hunk 1]
```diff
@@ -105,6 +106,10 @@ export class HomepageComponent implements OnInit, OnDestroy, AfterViewInit {
       return;
     }
 
+    if (carbonRef) {
+      sponsorsWrapper.classList.add('sponsors-carbon');
```
Comment (kamilmysliwiec) [inline src/app/homepage/homepage.component.ts:110]: When is this class removed (upon what conditions)? Like, what if you resize a window?

Comment (Arhell) [reply src/app/homepage/homepage.component.ts:110]: If there is a banner then it is there and it appears, if not then no

Comment (Arhell) [reply src/app/homepage/homepage.component.ts:110]: after fix, before in issue image https://github.com/nestjs/docs.nestjs.com/issues/1790

**Comment by kamilmysliwiec** [comment]
LGTM

**Pull request merged**